ihmc-common-walking-control-modules/src/main/java/us/ihmc/commonWalkingControlModules/highLevelHumanoidControl/MPCHumanoidHighLevelControllerManager.java

```diff
@@ -0,0 +1,427 @@
+package us.ihmc.commonWalkingControlModules.highLevelHumanoidControl;
```

Comment:
I'm not sure this needs to exist.  Is there anything currently different from the regular one?

Reply:
There are a few differences. 
1. MPC one has input value of desiredMPCControllerOutput
2. In the copyJointDesiredToJoints, MPC one populates the mpcControllerOutput, instead regular one populate the wholebodyControllerCoreOutput.

The second is key point, and first one comes from second one